package.json

```diff
@@ -15,6 +15,7 @@
     "format-prettier": "prettier -w 'static/js/*.{js,jsx,ts,tsx}' 'static/sass/*.scss'",
     "lint-python": "flake8 --extend-ignore=E203 webapp tests && black --check --line-length 79 webapp tests",
     "lint-scss": "stylelint static/**/*.scss",
+    "lint-jinja": "djlint $CHANGED_FILES --lint || true",
```

Comment:
This will not run locally right? Because `$CHANGED_FILES` is only set on CI?

Maybe it doesn't have to run locally, if you don't want or don't need it to, but in such case, maybe it's not needed in package.json at all?
If it depends on `$CHANGED_FILES`, can only be run on CI, just run it directly in CI job. Because having it in package.json suggests it's something that can be run locally in dev environment, which currently it can't.

Also, why is the `|| true` needed?

Reply:
yes, makes total sense @bartaz. I removed it from package.json.
And I don't know what `|| true` is for, it was in Andreu's similar yarn script, so I just copied it :D

Reply:
Locally it's not needed because djlint will automatically format the code on save and will highlight any errors in files.